yapapi/agreements_pool.py

```diff
@@ -6,6 +6,8 @@
 import sys
 from typing import Dict, NamedTuple, Optional, Set, Tuple
 
+import aiohttp
```

Comment:
I'd move the import tot the top

Reply:
I'm following PEP 8 here (https://www.python.org/dev/peps/pep-0008/#imports):
```
Imports should be grouped in the following order:

    Standard library imports.
    Related third party imports.
    Local application/library specific imports.

You should put a blank line between each group of imports.
```

Reply:
oh my ...